### DISCUSSION TITLE
New Update

### DISCUSSION THREAD

**Discussion by Directox01** [virtio-win/kvm-guest-drivers-windows#928] [General]
Hello, May I ask if there are plans to release a new update coming days? Thank you.

**Comment by vrozenfe** [comment]
Yes, I do it soon.
Vadim.

**Reply by Directox01** [reply]
Thank you <3

**Comment by Directox01** [comment]
I would like to ask once again if there is an ETA for the next release of the 0.249 version. Thank you.

**Comment by vrozenfe** [comment]
It's all ready at our side. Just need to fix the issue raised at https://github.com/virtio-win/kvm-guest-drivers-windows/issues/975 
Sorry for the inconvenience.
Vadim.

**Comment by Directox01** [comment]
Thank you  for the information!

**Comment by Directox01** [comment]
Hello! Do we have an ETA for the new version? Thanks!

**Comment by vrozenfe** [comment]
Last decade of February. 
Maybe a bit earlier.
Vadim.

**Reply by Directox01** [reply]
Thx !

**Comment by Directox01** [comment]
Greetings! any excepted stable update ( release ) at coming days? Thank you once again.

**Comment by vrozenfe** [comment]
Hopefully by the end of May or so.
Best,
Vadim.

**Comment by Directox01** [comment]
Good luck. Thank you buddy!

**Comment by Directox01** [comment]
Any news yet? :)

**Comment by vrozenfe** [comment]
We have a new release available at https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.262-2

but it probably has some issues we are trying to reproduce and investigate https://github.com/virtio-win/virtio-win-guest-tools-installer/issues/64

**Comment by Directox01** [comment]
Thank you!

**Comment by lixianming19951001** [comment]
@vrozenfe 
"Hello! I hope you’re doing well. Could you please let me know when we might expect a new release version? Thank you very much for your assistance!"
Best regards, Li.

**Comment by vrozenfe** [comment]
@lixianming19951001 

Hopefully, by the and of this week.

Best,
Vadim.

**Reply by lixianming19951001** [reply]
@vrozenfe 
I hope this message finds you well. I apologize for any inconvenience, but I noticed that two new tags, mm282 and mm286, were added today. The commit hash for mm286 is 96b358e, but I am particularly interested in the commit titled [viostor] Backport vioscsi SendSRB fixes and improvements (commit hash: 379f291), which comes after mm286.

Could you please let me know when we can expect a new release that includes the mentioned commit?

Thank you very much for your assistance!

Best，
Li.

**Comment by Directox01** [comment]
Hello, any ETA when we can see a fully compatible version for win 2025? Thank you.
